src/Views/Form/Templates/Classic/resources/js/form.js

```diff
@@ -0,0 +1,99 @@
+/** @jsx h */
+/** @jsxFrag h */
+import h from 'vhtml';
+import {domIsReady, insertAfter, nodeFromString, removeNode} from './not-jquery.js';
+
+// Transforms document for classic template
+domIsReady(() => {
+	removeTestModeMessage();
+	movePersonalInfoSectionAfterDonationAmountSection();
+	movePaymentFormInsidePaymentDetailsSection();
+	setPersonalInfoTitle();
+	addPersonalInfoDescription();
+	setPaymentDetailsTitle();
+	addPaymentDetailsDescription();
+	splitDonationLevelAmountsIntoParts();
+});
+
+/**
+ * Individual transformations
+ */
+
+function removeTestModeMessage() {
+	removeNode(document.querySelector('#give_error_test_mode')); // Get out of my way!
```

Comment:
There is an error in the console pointing to this line 

![image](https://user-images.githubusercontent.com/4222590/139807548-1ba65f16-356f-4214-9067-51df9db7f2a6.png)

Reply:
Ah, I just need to make that conditional. That whole thing was temporary anyway.